buildSrc/build.gradle.kts

```diff
@@ -1,3 +1,14 @@
 plugins {
     `groovy-gradle-plugin`
 }
+
+repositories {
+    gradlePluginPortal()
+    mavenCentral()
+}
+
+dependencies {
+    implementation("io.micronaut.gradle:micronaut-gradle-plugin:3.5.1")
+    implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.10")
+    implementation("org.jetbrains.kotlin:kotlin-allopen:1.6.0")
```

Comment:
@melix I couldn't find a way to use the version catalog here. Is it possible at all?

Reply:
Yes, you can if you create a `settings.gradle` file in `buildSrc` and import the same catalog as the top level project. See https://docs.gradle.org/current/userguide/platforms.html#sec:importing-catalog-from-file

Reply:
Great, that was the secret sauce, thanks!